assets/blocks/editor-components/limited-text-control/index.test.js

```diff
@@ -55,4 +54,127 @@ describe( '<LimitedTextControl />', () => {
 
 		expect( queryByText( 'Characters: 10/20' ) ).toBeTruthy();
 	} );
+
+	it( 'Should respect the maxLength property', async () => {
+		const onChangeMock = jest.fn();
+		const { queryByRole } = render(
+			<LimitedTextControl
+				value="ABC"
+				maxLength={ 3 }
+				onChange={ onChangeMock }
+			/>
+		);
+
+		await userEvent.type( queryByRole( 'textbox' ), 'DEF' );
+
+		expect( onChangeMock ).toHaveBeenCalledTimes( 0 );
+	} );
+
+	it( 'Should ignore new lines', async () => {
+		const onChangeMock = jest.fn();
+		const { queryByRole } = render(
+			<LimitedTextControl
+				value=""
+				maxLength={ 20 }
+				onChange={ onChangeMock }
+			/>
+		);
+
+		const element = queryByRole( 'textbox' );
+
+		await userEvent.type( element, '{enter}' );
+
+		expect( onChangeMock ).toHaveBeenCalledTimes( 0 );
+	} );
+} );
+
+describe( '<LimitedTextControl multiline="{ true }"/>', () => {
```

Comment:
In the same sense, do we need the quotes here? 

Testing here, this works:

```suggestion
describe( '<LimitedTextControl multiline={ true }/>', () => {
```